haptics/android/src/main/java/com/capacitorjs/plugins/haptics/arguments/HapticsImpactType.java

```diff
@@ -0,0 +1,43 @@
+package com.capacitorjs.plugins.haptics.arguments;
+
+public enum HapticsImpactType implements HapticsVibrationType {
+    LIGHT("LIGHT", new long[] { 0, 50 }, new int[] { 0, 110 }, new long[] { 0, 20 }),
+    MEDIUM("MEDIUM", new long[] { 0, 43 }, new int[] { 0, 180 }, new long[] { 0, 43 }),
+    HEAVY("HEAVY", new long[] { 0, 60 }, new int[] { 0, 255 }, new long[] { 0, 61 });
+
+    private final String mType;
```

Comment:
I think we should abandon Hungarian notation in our plugins. Its kinda old and Google advises against it now in their Java style guide https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names